functests/dpdk/dpdk.go

```diff
@@ -581,6 +581,68 @@ func ValidateSriovPolicy() (bool, error) {
 	return false, nil
 }
 
+func findPodByNodeFromList(nodeName string, podList []corev1.Pod) (corev1.Pod, error) {
+	for _, pod := range podList {
+		if pod.Spec.NodeName == nodeName {
+			return pod, nil
+		}
+	}
+
+	return corev1.Pod{}, fmt.Errorf("failed to find pod on node %s", nodeName)
+}
+
+// FindDpdkSriovDevice will search for the default gateway interface to be used as the PF
+func FindDpdkSriovDevice(enabledNodes *sriovcluster.EnabledNodes, nodeName string) (*sriovv1.InterfaceExt, error) {
+	s, ok := enabledNodes.States[nodeName]
```

Comment:
nodeStatus. You are using s way below, where it's harder to get its context

Reply:
+1